cli/flags/common.go

```diff
@@ -69,6 +69,8 @@ func (commonOpts *CommonOptions) InstallFlags(flags *pflag.FlagSet) {
 	flags.Var(opts.NewQuotedString(&tlsOptions.CertFile), "tlscert", "Path to TLS certificate file")
 	flags.Var(opts.NewQuotedString(&tlsOptions.KeyFile), "tlskey", "Path to TLS key file")
 
+	flags.Var(opts.NewQuotedString(&commonOpts.TrustKey), "key-path", "Path to key file for ID and image signing")
```

Comment:
Also; this is in common flags, so this allows me to use;

```
docker --key-path=/foobar info
```

But doing so doesn't make a difference for the `ID` printed in `docker info`